### ISSUE TITLE
Build a throwaway AppClip target

### ISSUE THREAD

**Issue by hboon** [AlphaWallet/alpha-wallet-ios#3793]
* Figure out what's the current uncompressed app size
* How far we need to slim down
* How much effort if we maintain it (i.e. refactor the app to get it working and any effect on future work)

**Comment by eviltofu** [comment]
@hboon What does the App Clip do?

**Comment by hboon** [comment]
Ultimately? Not exactly sure.

See how much of the app can fit in and what doesn't? Is it code and frameworks (which means we have to drop functionality) or is it images (maybe we can resize them or load on demand).

**Comment by eviltofu** [comment]
Variant: AlphaWallet.ipa
Supported variant descriptors: Universal
App + On Demand Resources size: 20.3 MB compressed, 50.6 MB uncompressed
App size: 20.3 MB compressed, 50.6 MB uncompressed
On Demand Resources size: Zero KB compressed, Zero KB uncompressed

**Comment by eviltofu** [comment]
Again, what does this thing do? Load Wallets and display the total sum of the wallet?

**Comment by hboon** [comment]
Ideally everything. Is there anything we can do to trim it down without losing functionality?

**Comment by eviltofu** [comment]
> Ideally everything. Is there anything we can do to trim it down without losing functionality?

@hboon We can't put the entire app into an app clip. We have to choose one functionality to implement.

**Comment by hboon** [comment]
Can you give try removing something arbitrary and form a theory to guide how much/how a feature contributes to space so we can see is the constraint?

**Comment by eviltofu** [comment]
> Can you give try removing something arbitrary and form a theory to guide how much/how a feature contributes to space so we can see is the constraint?

From the Apple documentation - ```An App Clip is a small part of your app that’s discoverable at the moment it’s needed and lets people complete a quick task from your app — even before installing your full app.```

I would suggest being able to select a wallet and view it (if we can access the saved wallets from the clip).

**Comment by hboon** [comment]
> From the Apple documentation - An App Clip is a small part of your app that’s discoverable at the moment it’s needed and lets people complete a quick task from your app — even before installing your full app.

Yes, but I would like to know how far we can push this so we know the constraints and can discover what we can do with it. If we can fit the entire app, that's the best and easiest to maintain, otherwise, how much do we have to remove and how can we estimate it?
